web/src/copy_and_paste.ts

```diff
@@ -360,12 +360,29 @@ function image_to_zulip_markdown(
     // Using Zulip's link like syntax for images
     return src ? "[" + title + "](" + src + ")" : (node.getAttribute("alt") ?? "");
 }
-
 function within_single_element(html_fragment: HTMLElement): boolean {
+    // Filter out non-element nodes by checking if the node is an HTMLElement or TextNode with non-whitespace content
+    const validElements = [...html_fragment.childNodes].filter((node) => {
+        if (node instanceof HTMLElement) {
+            return node.nodeName !== "BR" && node.nodeName !== "META";
+        } else if (node.nodeType === (globalThis.Node ? globalThis.Node.TEXT_NODE : 3)) {
```

Comment:
Why is the 3 here?

Reply:
The 3 is used as a fallback value for the text node type in case globalThis.Node.TEXT_NODE is unavailable. In the DOM, node types are represented by numeric constants:

1: Represents an element node (e.g., <div>, <p>)
3: Represents a text node
So, 3 is the standard numeric value for a text node.